docs/kafka/kafka-bin-scripts-kafka/README.adoc

```diff
@@ -129,25 +126,23 @@ endif::[]
 == Configuring the Kafka scripts to connect to a Kafka instance
 
 [role="_abstract"]
-To enable the Kafka scripts to access a Kafka instance, you must configure the connection using the generated credentials for your {product-kafka} service account. For the Kafka scripts, you will create a configuration file that defines these values.
+To enable the Kafka scripts to access a Kafka instance, you must configure a connection using the generated credentials for your {product-long-rhoas} service account. In this task, you create a configuration file that defines these credential values.
 
 .Prerequisites
-ifndef::qs[]
-* You have the generated credentials for your service account. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
-* You've set the permissions for your service account to access the Kafka instance resources. To verify the current permissions, select your Kafka instance in the {product-kafka} web console and use the *Access* page to find your service account permission settings.
-endif::[]
+* You have the generated credentials for your service account. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page.
+* You've set permissions for your service account to access resources in the Kafka instance. To verify the current permissions, select your Kafka instance in the {product-kafka} web console and click the *Access* tab. To learn more about setting permissions, see {base-url}{access-mgmt-url-kafka}[Managing account access in {product-long-kafka}^].
```

Comment:
Refer to the console with its long name on first usage in assembly or module.

Reply:
Woops, yes - thank you!